nuc_morph_analysis/analyses/neighbor_of_X/example_dying_track_neighbors.py

```diff
@@ -0,0 +1,89 @@
+#%%
+from nuc_morph_analysis.lib.visualization.notebook_tools import save_and_show_plot
+from nuc_morph_analysis.lib.preprocessing import global_dataset_filtering
+import numpy as np
+from pathlib import Path
+import os
+import matplotlib.pyplot as plt
+from nuc_morph_analysis.lib.preprocessing import load_data
+from nuc_morph_analysis.lib.preprocessing import labeling_neighbors_helper
+from nuc_morph_analysis.lib.visualization.plotting_tools import plot_colorized_img_with_labels
+
+# TEMP: loading local for testing and speed
+df = global_dataset_filtering.load_dataset_with_features(dataset='all_baseline',load_local=True)
+#%%
+# for testing only use a subset of timepoints
+track_id = 87124
+
+TIMEPOINT = int(df.loc[df['track_id']==track_id,'identified_death'].values[0])
+frames_before = 3
+frames_after = 3
```

Comment:
These could be capital for consistency with hard coded features at the top

Reply:
done